### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by charlottekostelic**
Updated dependencies for requests, black, pytest, pytest-cov, and pytest-mock. Updated tests with new path to credentials, new version number, and updated control numbers.

### CODE REVIEW

File: .github/workflows/unit-tests.yaml [hunk 1]
```diff
@@ -5,8 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
```
Comment (klinga) [inline .github/workflows/unit-tests.yaml:8]: Did 3.7 cause any issues?

**Review by klinga — APPROVED** [review]
Good job 👏

File: .github/workflows/unit-tests.yaml (hunk 1 above)
Comment (charlottekostelic) [reply .github/workflows/unit-tests.yaml:8]: I removed 3.7 after the tests failed.

Comment (klinga) [reply .github/workflows/unit-tests.yaml:8]: hmmm, let me check logs to see what exactly failed

Comment (klinga) [reply .github/workflows/unit-tests.yaml:8]: Oh, I see, this is related to deprecated functionality in github actions, not necessarily our code. It makes sense to move forward, we don't have to provide any backward compatibility here and complicate how actions are run

**Pull request merged**